### ISSUE TITLE
Make it possible for UMC to use HelpWML (GNA #13338)

### ISSUE THREAD

**Issue by wesnoth-bugs** [wesnoth/wesnoth#1155]
**[Original submission](https://gna.org/bugs/index.php?13338)** by espreon on 2009-04-06

Sometimes, UMCs use custom traits, and as the help system does not generate entries for traits as it does for things like abilities, it would be nice to allow UMCs to be able to use HelpWML.

(Reproduced on Ubuntu Linux)
Release: 1.7+svn
Priority: 3 - Low
Severity: 1 - Wish

**Comment by wesnoth-bugs** [comment]
**Modified on 2009-07-08**

**ancestral wrote:**

> This would actually be exceptional, because you could make the Units section in Help independent for each campaign. This way we could actually control the encountered units correctly per campaign.
> 
> 
> 
> Currently, if you were to play HttT and then do a multiplayer game with Drakes, when you go back to playing HttT you'd see Drakes in the Units section of Help. If this were independent, you would *not* see Drakes since you haven't encountered them in the game.
> 
> 
> 
> (related to #13908)

**Comment by CelticMinstrel** [comment]
Waiting on #994.

**Comment by jostephd** [comment]
#994 has been replaced by #3653.

**Comment by Wedge009** [comment]
#3653 is merged.

**Comment by babaissarkar** [comment]
Duplicate of #2376.
